common/djangoapps/track/middleware.py

```diff
@@ -137,6 +138,7 @@ def enter_request_context(self, request):
             'session': self.get_session_key(request),
             'user_id': self.get_user_primary_key(request),
             'username': self.get_username(request),
+            'ip': get_ip(request),
```

Comment:
Not being too familiar with `django-ipware`/our networking situation, are there times when this could return `None`, and is that OK if so?